tests/codeception/acceptance/Content/NewsCest.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+/**
+ * Test the news functionality.
+ */
+class NewsCest {
+
+  /**
+   * Test that the default content has installed and is unpublished.
+   */
+  public function testDefaultContentExists(AcceptanceTester $I) {
+    $I->logInWithRole('administrator');
+    $I->amOnPage("/admin/content");
+    $I->see("Sample: Smith Conference");
+    $I->see("Sample: For Runners, Is 15 Feet the New 6 Feet for Social Distancing?");
+    $I->see("Sample: Stanford researchers find that misfiring from jittery neurons");
```

Comment:
when we get alot of default content, these might be on a different pager.
something we might have to deal with later

Reply:
yeah.